### ISSUE TITLE
bad performance for large batch size

### ISSUE THREAD

**Issue by Kindpire** [open-mmlab/mmdetection#1914]
Hi, I trained an SSD model with configs/pascal_voc/ssd300_voc.py, only changed the batch size from 8 to 64. I know the original paper used 32 (8 images for 4 GPUs). the final mAP dropped from 77.5 to 71.8. any idea about how to keep the good result with a large batch size?

**Comment by hellock** [comment]
Did you modified the learning rate accordingly?

**Comment by Kindpire** [comment]
> Did you modified the learning rate accordingly?

Thx, yes. I'm adjusting the learning rate now. The mAP improves a lot to 75 after changed lr to 2e-3.
